src/chi/components/range-slider/range-slider.scss

```diff
@@ -160,7 +118,7 @@ input {
       &::-webkit-slider-thumb {
         @include thumb-style;
         -webkit-appearance: none;
-        margin-top: -6px;
+        margin-top: -8px;
```

Comment:
@mattnickles can these px measures be converted to rems?

Reply:
Changed them all. Thanks!